src/osha/oira/statistics/utils.py

```diff
@@ -173,15 +210,32 @@ def assessment_rows(offset):
                     account_id=account.id,
                     account_type=account.account_type,
                 )
-                for session, account, total_risks, answered_risks in batch
-            ]
-            return rows
+                if existing:
+                    for name, value in attribs.items():
+                        setattr(existing, name, value)
+                else:
+                    self.session_statistics.add(SurveySessionStatistics(**attribs))
+                handled = handled + 1
+            return handled
 
-        log.info("Table: assessment")
         self._process_batch(assessment_rows)
 
     def update_account(self, country=None):
-        accounts = self.session_application.query(Account).order_by(Account.id)
+        log.info("Table: account")
+        latest_creation_date = (
+            self.session_statistics.query(
+                sqlalchemy.func.max(AccountStatistics.creation_date)
+            ).first()[0]
+            or datetime.min
+        )
+        if latest_creation_date > datetime.min:
+            log.info(f"Skipping accounts up to and including {latest_creation_date}")
```

Comment:
The same comments I did before about the use of first and the f-string also apply here